### ISSUE TITLE
allow drag and drop to RPC window

### ISSUE THREAD

**Issue by dnai05** [keepassxreboot/keepassxc#10477]
**Background**: for work I mostly use KPXC to log into RPC windows. This means I have to use local auto-type with the risk of login details being typed into the incorrect window. I am using dual monitors.

**Suggestion**: don't know if this is possible but could you add the option to drag and drop user credentials to a RPC window. The drag would transfer the details based on the autotype sequence. When you drop onto login window it would fill the appropriate fields. Alternatively the drop pops up options like Username + Password, Username only, Password only and types them into the appropriate fields. I also suggest that this option stops KPXC from minimizing so its easier to login to multiple servers.

**Benefit**: able to target required window directly and not rely on OS to select window. 

KeePassXC - Version 2.7.7
Revision: 68e2dd8

Qt 5.15.11
Debugging mode is disabled.

Operating system: Windows 11 Version 2009
CPU architecture: x86_64
Kernel: winnt 10.0.22621

Enabled extensions:
- Auto-Type
- Browser Integration
- Passkeys
- SSH Agent
- KeeShare
- YubiKey
- Quick Unlock

Cryptographic libraries:
- Botan 3.1.1

**Comment by Tipoff4317** [comment]
Yeah, I would be nice to be drag-and-dropping at least the password to another field/windows in Windows as well.

**Comment by digix77** [comment]
Hope the drag and drop from KeepassXC entries will be added in one of the future releases.

**Comment by Starfoxfs** [comment]
I used Keepass before under Linux and it already can Drag&Drop Username or Password to another Window (like an open Firefox).

KeepassXC works nice with Autotype now on Linux but this would be a huge improvement if it can Drag & Drop the Username & Password to another Window or App.
